src/dmd/dinterpret.d

```diff
@@ -4997,6 +5000,24 @@ public:
                     result = CTFEExp.voidexp;
                 return;
             }
+            else if (fd.ident == Id._d_arraysetlengthT)
```

Comment:
Please add an explanation for why this logic is needed.  You game me a good explanation on Slack (if I remember correctly).  Maybe you could just copy that to here.

Reply:
Will do

Reply:
Better now?

Reply:
I see now the explanation on line 5008.  I'm sorry, but I didn't see it earlier.  No other explanation is needed, IMO.  I think you put it in the wrong spot anyway ` if (fd.ident == Id.__ArrayPostblit || fd.ident == Id.__ArrayDtor)`